### ISSUE TITLE
Mini Cart block button should be disabled in the editor

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#5274]
## To reproduce
1. Create a post or page.
2. Add a Mini Cart block.
3. Click on the Mini Cart button.
4. Notice the block border is not shown (blue). You need to click outside of the button to make it appear.

https://user-images.githubusercontent.com/3616980/143916178-f58da793-38d7-41bc-91b3-73a18fb5521b.mp4

## Expected behavior
The block border should become visible when clicking on the Mini Cart button.

I think that can be fixed setting the button to disabled. Ideally, we should use the [`Noninteractive` component](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5157) so the style is not affected.
